docs/designdoc/clustering_stop.md

```diff
@@ -71,27 +74,69 @@ status:
       lastTransitionTime: 2018-01-01T00:00:00Z
 ```
 
-Also, `STOPPED` will be added to the table displayed when you do `kubectl get mysqlcluster`.
+Also, `CLUSTERING STOPPED` will be added to the table displayed when you do `kubectl get mysqlcluster`.
 
 ```console
 $ kubectl get mysqlcluster
-NAME   AVAILABLE   HEALTHY   STOPPED   PRIMARY   SYNCED REPLICAS   ERRANT REPLICAS   LAST BACKUP
-test   Unknown     Unknown   True      0         2                                   <no value>
+NAME   AVAILABLE   HEALTHY   PRIMARY   SYNCED REPLICAS   ERRANT REPLICAS   CLUSTERING STOPPED   RECONCILE STOPPED   LAST BACKUP  
+test   Unknown     Unknown   0         2                                   True                                     <no value>
 ```
 
-When you execute `kubectl moco clustering start <CLSUTER_NAME>`,
+When you execute `kubectl moco start clustering <CLSUTER_NAME>`,
 `moco.cybozu.com/clustering-stopped: "true"` is removed from `.metadata.annotation` of MySQLCluster.
 
-The controller resumes the reconcile process for `MySQLCluster` that does not have the `moco.cybozu.com/clustering-stopped: "true"` annotation.
+The controller resumes the `ClusterManager` for `MySQLCluster` that does not have the `moco.cybozu.com/clustering-stopped: "true"` annotation.
 When the `ClusterManager` resumes operation, the `.status` of `ClusteringStopped` in `.status.conditions` is updated to `False`,
 and the `.status` of other `.status.conditions` is also updated from `Unknown`.
 
+### Reconciliation
+
+When you execute `kubectl moco stop reconciliation <CLSUTER_NAME>`,
+the `moco.cybozu.com/reconciliation-stopped: "true"` is added to the `.metadata.annotation` of MySQLCluster.
+
+```yaml
+metadata:
+  annotations:
+    moco.cybozu.com/reconciliation-stopped: "true"
+```
+
+The reconcile process will not be executed for MySQLCluster resources annotated with this annotation.
+Therefore, even if changes are made to the fields of the MySQLCluster resource, changes will not be propagated to other resources.
+
+The only exception is the stopping and restarting of clustering by the `moco.cybozu.com/clustering-stopped` annotation.
+Also, the `ClusterManager` does not stop even if the reconcile process is stopped.
+
+The `ReconciliationStopped` gets added to `.status.conditions` of MySQLCluster.
+
+```yaml
+status:
+  conditions:
+    - type: ReconciliationStopped
+      status: "True"
+      lastTransitionTime: 2018-01-01T00:00:00Z
```

Comment:
When a MySQL cluster is normal, I want all ".status.conditions" to be "true".
Could you rename the type of the condition to reverse the boolean value?

Also, please rename the following names.

- Header of `kubectl get mysqlcluster`
- Annotation key
- Metrics name